index.js

```diff
@@ -102,7 +107,11 @@ module.exports = function(options) {
   var scopeTest = new RegExp('^@');
   var scopeDecomposition = new RegExp('^@(.+)/(.+)');
 
-  multimatch(names, pattern).forEach(function(name) {
+  var multimatched = multimatch(names, pattern);
+  logDebug('multimatch pattern: ' + pattern);
+  logDebug('multimatched: ' + multimatched);
```

Comment:
Yeah! But it was very helpful to understand what was happing, cause the error occurs only if you change the default pattern.
Anyway, I will remove this logs ;)